src/mercury_engine_data_structures/formats/bctex.py

```diff
@@ -86,3 +91,97 @@
         level=9,
     ),
 )
+
+class PICATextureFormat(Enum):
```

Comment:
also just noticed, out of curiosity, why is this called `PICATextureFormat` instead of `CTPKTextureFormat`?

Reply:
Guess because the format is proprietary to the PICA2000 GPU used in the 3DS and doesn't necessary needs to be in a CTPK file.